static/js/admin/src/AddMenu.js

```diff
@@ -9,19 +9,56 @@ define('AddMenu', [ 'jquery', 'jquery-ui', 'underscore', 'CreateContainerForm',
 	
 	AddMenu.prototype.getMenuItems = function() {
 		var items = {};
+
 		if ($.inArray('addRemoveContents', this.container.permissions) != -1) {
-			items["addContainer"] = {name : "Add Container"};
-			items["ingestPackage"] = {name : "Add Ingest Package"};
-			items["ingestSource"] = {name : "Add from File Server"};
-			items["simpleObject"] = {name : "Add Simple Object"};
-			
+			var self = this;
+
+			items["addContainer"] = {
```

Comment:
Are you sure that is the case?  Because if so, if you switched pages between a folder you didn't have permission on to one you did, you wouldn't get any entries in the add menu because of the preceeding if statement.  If so, then that needs to be fixed too, but I would like to verify first.

The isVisible method is one that you defined further down as a member function of the AddMenu prototype, so you should be able to name it something else.  I understand that the `visible` property of the menu entry is reserved though